packages/web-twig/newVersion.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This file is not necessary ;)

Reply:
@janicekt Why is it not necessary?

Reply:
I will rearrange the release process in next PRs.